modules/extra/mysql.cpp

```diff
@@ -248,13 +249,14 @@ class ModuleSQL final
 				const Anope::string &user = block->Get<const Anope::string>("username", "anope");
 				const Anope::string &password = block->Get<const Anope::string>("password");
 				unsigned int port = block->Get<unsigned int>("port", "3306");
+				const Anope::string &socket = block->Get<const Anope::string>("socket", "");
 
 				try
 				{
-					auto *ss = new MySQLService(this, connname, database, server, user, password, port);
+					auto *ss = new MySQLService(this, connname, database, server, user, password, port, socket);
 					this->MySQLServices.emplace(connname, ss);
 
-					Log(LOG_NORMAL, "mysql") << "MySQL: Successfully connected to server " << connname << " (" << server << ")";
+					Log(LOG_NORMAL, "mysql") << "MySQL: Successfully connected to server " << connname << " (" << (socket.empty() ? server + ":" + port : socket) << ")";
```

Comment:
Just to check is this acceptable, or should I have duplicated the line like it did for the debug logging at the end. (also should the debug logging at the end be like this instead?)

Reply:
Its fine like this.